charts/adminer/values.yaml.gotmpl

```diff
@@ -49,21 +49,19 @@ ingress:
   enabled: true
   className: ""
   annotations:
-    cert-manager.io/cluster-issuer: "letsencrypt-production"
+    cert-manager.io/cluster-issuer: "cert-issuer"
   tls:
     - hosts:
-        - k8s.monitoring.${MACHINE_FQDN}
+        - k8s.monitoring.{{ requiredEnv "MACHINE_FQDN" }}
```

Comment:
Q: what is the advantage of gotmpl vs jinja2?

Reply:
we use `helmfile`. `helmfile` natively supports `values.yaml.gotmpl`. If we used `jinja2`, then we would have to manually run it.